concepts/Common/.NET Backend Solutions/Web API Service (CRUD)/Web API Service (CRUD).md

```diff
@@ -0,0 +1,49 @@
+
+- **Multiple Authentication Strategies**     
+You application’s authentication mechanism can leverage built-in support for OAuth2, JWT, and OpenID. You can also implement a custom authentication strategy.
+
+- **Cross-Platform Compatibility**     
+Backend Web API Service is platform-agnostic. You can access user and role storage, permission management, authorization, reporting, and other functionality from various types of clients – web, mobile, and desktop applications.
+
+- **IDE Integration**    
+If you work in Visual Studio 2022+ for Windows, integrated Solution Wizard automatically scaffolds your Web Service to speed up application development.
+
+Basic functionality of DevExpress Web API Service is available for free. Additional capabilities and services listed below are available as part of the [DevExpress Universal Subscription](https://www.devexpress.com/subscriptions/universal.xml):
+
+- Audit trail
+- Endpoints to download reports and file attachments
+- Data validation
+- Localization endpoints that return translated captions for UI elements
+- Technical support and full source code
+
+#### Get Started: Add DevExpress Web API Service Functionality
+
+Refer to the following articles for guidance on how to get started:
+
+- [Predefined Users, Roles, and Permissions](https://docs.devexpress.com/eXpressAppFramework/119065/data-security-and-safety/security-system/security-object-model/predefined-users-roles-and-permissions): describes how to configure permissions for your ORM data models or your EF Core entities.
+
+- [JavaScript (DevExtreme) Example](https://go.devexpress.com/XAF_Security_NonXAF_DevExtreme_OData.aspx): shows a client-side HTML/JavaScript CRUD app that uses the DevExtreme Data Grid and connects to an OData v4 web service (using the ASP.NET Core Web API).
+
+- [ASP.NET Core MVC Example](https://www.devexpress.com/go/XAF_Security_NonXAF_MVC_Core.aspx): shows a server-side ASP.NET MVC Core CRUD app that uses our high-performance DevExtreme-based Data Grid. The demo supports inline data editing.
+
+- [JavaScript (Svelte) Example](https://community.devexpress.com/blogs/news/archive/2023/04/11/consume-the-devexpress-backend-web-api-from-javascript-with-svelte-part-1.aspx): shows how to build a Svelte app and consume CRUD and custom Web API Service endpoints to authenticate users and protect data, edit and validate data, localize captions, preview and download reports.
+
+#### Authorization (Role-Based Access Control)
+
+XAF UI and Web API Service apps ship with built-in Role-based Access Control (RBAC) - a part of the [Security System module](https://docs.devexpress.com/eXpressAppFramework/113366/data-security-and-safety/security-system). We recommend that you configure Type, Record, and Field level permissions for different user roles. This way, you implement proper data and UI authorization support (filter sensitive data automatically and authorize CRUD/custom operations).
```

Comment:
> XAF UI

It's out of the context. We're talking about Web API Service, so let's remove the XAF UI entry.